cmake/OpenCVDetectVTK.cmake

```diff
@@ -1,3 +1,21 @@
+# VTK 9.0
+find_package(VTK 9 QUIET NAMES vtk COMPONENTS
+    FiltersExtraction
+    FiltersSources
+    FiltersTexture
+    IOExport
+    IOGeometry
+    IOPLY
+    InteractionStyle
+    RenderingCore
+    RenderingLOD
+    RenderingOpenGL2
+    NO_MODULE)
+if(VTK_FOUND)
+  set(HAVE_VTK ON)
+  return()
```

Comment:
I believe we still need `try_compile()` test with `checks/vtk_test.cpp` - see below.

Reply:
Done